### DISCUSSION TITLE
Result interpretation - DInSAR technique

### DISCUSSION THREAD

**Discussion by RenzoAnc** [gmtsar/gmtsar#456] [Q&A] [answered]
Hello everyone. I'd like to ask a basic question about displacements obtained with DInSAR using GMTSAR. 

If I use as master image an older image (for example, 18/01/2018), and slave image a newer image (for example, 30/01/2018), should I interpret the results as the following?:

Images with ascending orbits:

Positive displacements:  it's oncoming in LOS direction
Negative displacements: it's moving away in LOS direction

Images with descending orbits:

Positive displacements: it's moving away in LOS direction
Negative displacements: it's oncoming in LOS direction

Thanks in advance.

**Comment by Xiaohua-Eric-Xu** [comment, marked as answer]
The positive and negative definition does not depend on orbit directions. For LOS displacement (i.e. los*), positive is moving toward the satellite. For phase (i.e. phase.grd, phasefilt.grd and unwrap.grd), positive is away from the satellite (i.e. increase in distance between satellite and ground, or range).

**Reply by RenzoAnc** [reply]
Thank your for your reply, professor Xiaohua. Sorry for the late reply. I've founded this information about this topic: 

https://topex.ucsd.edu/SV_7.1/index.html

, and now I understand the difference of the results when I use images with different orbits.

Positive values: Moving toward the satellite 

Ascending orbits: westward move
Descending orbits: eastward move

Negative values: Moving away from the satellite 

Ascending orbits: eastward move
Descending orbits: westward move

I have another question about SBAS processing. I've read about a "pin" I need to choose to get better displacements results with SBAS. Is it just used to improve the results founded with the inversion problem?.  This step is necessary or it can be optional. I don't understand much about the importance of it. 

I'll be thankful to you if you explain me a bit more about it.

Regards.

**Reply by Xiaohua-Eric-Xu** [reply]
There could be also up and downs which are the same for both orbits. 

If you are targeting an accurate time-series measurement, this is kind of necessary. Or if your area of interest is small, you could also detrend your data. To make a pin, https://github.com/gmtsar/gmtsar/issues/345#issuecomment-1183515053

**Reply by RenzoAnc** [reply]
Thank you, professor @Xiaohua-Eric-Xu. My AOI is small (landslide), but it's surrounded by mountains and others which can slide in certain periods of time. So, I don't know if detrending will be necessary. About the pin, It needs to be a position which doesn't change so much along the time, doesn't it?. I was thinking to choose an infrastructure of a city as the landslide is near it.

Regards, professor.

**Reply by Xiaohua-Eric-Xu** [reply]
Yes, something stable and with strong reflectivity.

**Reply by RenzoAnc** [reply]
Thank you, professor @Xiaohua-Eric-Xu.

**Reply by omarwater98** [reply]
> Yes, something stable and with strong reflectivity.

Hi, teacher for make a DInSAR process i have to select (phase difference) the shift_topo it would be 1 too?

#####################################
#   parameters for make topo_ra     #
#   - dem2topo_ra.csh               #
#####################################
# subtract topo_ra from the phase
#  (1 -- yes; 0 -- no)
topo_phase = 1
# if above parameter = 1 then one should have put DEM.grd in topo/

# topo_ra shift (1 -- yes; 0 -- no)
shift_topo = 0

**Reply by RenzoAnc** [reply]
Hello, @omarwater98. I hope it to helps you: https://github.com/PaulWessel/gmtsar. You could revise the SHIFT THE TOPOPHASE [OPTIONAL] section for it. I'll be waiting for the professor @Xiaohua-Eric-Xu 's answer too.

Regards

**Reply by omarwater98** [reply]
> Hello, @omarwater98. I hope it to helps you: https://github.com/PaulWessel/gmtsar. You could revise the SHIFT THE TOPOPHASE [OPTIONAL] section for it. I'll be waiting for the professor @Xiaohua-Eric-Xu 's answer too.
> 
> Regards

In other words, if I don't generate the topo_shift.grd, a phase difference map is not created? Otherwise it would only be showing a digital elevation model? @Xiaohua-Eric-Xu

**Reply by Xiaohua-Eric-Xu** [reply]
I think topo_shift is only needed when you have a pair of data that's with very large baseline like ALOS-1. For other cases, it is usually not needed.
